examples/cli/src/main/java/io/unitycatalog/cli/TableCli.java

```diff
@@ -109,11 +110,13 @@ private static void handleTableStorageLocation(String storageLocation, List<Colu
         if (!storageLocation.startsWith("s3://")) {
             // local filesystem path
             Path path = getLocalPath(storageLocation);
-            if (!(Files.exists(path))) {
-                // try and initialize the directory and initiate delta log at the location
-                try {
-                    DeltaKernelUtils.createDeltaTable(path.toUri().toString(), columnInfos, null);
-                } catch (Exception e) {
+            // try and initialize the directory and initiate delta log at the location
+            try {
+                DeltaKernelUtils.createDeltaTable(path.toUri().toString(), columnInfos, null);
+            } catch (Exception e) {
+                if (e.getCause() != null && e.getCause().getClass().equals(TableAlreadyExistsException.class)) {
+                    // TODO confirm the schema of the existing table matches the schema of the new table
```

Comment:
@ravivj-db Can you create a GH issue for this TODO?

Reply:
Done : https://github.com/unitycatalog/unitycatalog/issues/82